test/verify/check-connection

```diff
@@ -88,6 +88,16 @@ class TestConnection(MachineCase):
         b.expect_load()
         b.enter_page("/system")
 
+        # Lets crash a process and see if we get a proper backtrace in the logs
+        # This helps with debugging failures in the tests elsewhere
+        if m.image in ["debian-unstable"]:
+            cron = "cron"
+        else:
+            cron = "crond"
+        m.execute("systemctl start {0}".format(cron))
```

Comment:
Because I don't want to start a whole new machine for it.

And yes given the variance and bullshit around how crashdumps in Linux (including SELinux issues) it's important that this works ... so that when we need it to diagnose a crash that happens 1 in 1000 times, on one of the operating systems, we have a backtrace and can try to fix it.